examples/dataflow-data-generator/data-generator-pipeline/data_generator/ParquetUtil.py

```diff
@@ -0,0 +1,41 @@
+import pyarrow as pa
```

Comment:
change request in original PR: https://github.com/GoogleCloudPlatform/professional-services/pull/199#discussion_r262758879